pkg/services/navtree/navtreeimpl/admin.go

```diff
@@ -18,16 +17,6 @@ func (s *ServiceImpl) getAdminNode(c *contextmodel.ReqContext) (*navtree.NavLink
 	orgsAccessEvaluator := ac.EvalPermission(ac.ActionOrgsRead)
 	authConfigUIAvailable := s.license.FeatureEnabled("saml")
 
-	if hasAccess(datasources.ConfigurationPageAccess) {
-		configNodes = append(configNodes, &navtree.NavLink{
-			Text:     "Data sources",
-			Icon:     "database",
-			SubTitle: "Add and configure data sources",
-			Id:       "datasources",
-			Url:      s.cfg.AppSubURL + "/datasources",
```

Comment:
Given this is a minor release I'm not sure if this should be deleted or the url changed to point to connections page so anyone that is used to finding the datasources list via admin navigation section can still get to it. @sympatheticmoose WDYT?

Reply:
Well, on the other hand if I click on something under the Admin section, I expect to land in the Admin section. Redirection at that point feels also very strange to me - not the best UX IMO.

Reply:
Both fair points. But:
- I don't think we want to wait till Grafana 11 to remove it
- We've at least advised that the data sources page was effectively deprecated with the warning banner
- If we changed the URL only for now, whilst its a more gentle removal, it feels similar to the initial redirect from clicking a datasource card.

Reply:
Thank you all for your inputs! I'll merge this then :)